app.js

```diff
@@ -0,0 +1,62 @@
+function addProduct(){
+    event.preventDefault();
+    const data = collectDataFromForm();
+    const newTR = generateTR(data);
+    injectTRToDOM(newTR);
+}
```

Comment:
missing call to `clearForm()`:
<img width="888" height="923" alt="Screenshot 2025-07-16 at 14 01 35" src="https://github.com/user-attachments/assets/c2512003-7cdf-425e-b228-3f8f176c2cef" />